src/platform/inventory/zephyr/inventory.c

```diff
@@ -25,6 +25,8 @@
 #include "log.h"
 #include "utils.h"
 
+#ifndef CONFIG_MENDER_CLIENT_INVENTORY_DISABLE
```

Comment:
This is not necessary as the whole file is omitted from the build in such a case, AFAIK. Should we check other files that might be completely depending on a config option and similar checks there?

Reply:
You are right, but it was there already for `src/inventory/inventory.c` and I wanted to keep it consistent. Do you prefer that I remove the other one instead?

I decided this way as it is harmless and if any, more defensive than the opposite (to remove it from `src/inventory/inventory.c`)

Reply:
We can leave the change to drop these in favor of skipping whole files to some cleanup PR. Not a biggie and as you say, it was already there. :+1: